inst/validation/bbr-requirements.yaml

```diff
@@ -614,6 +614,10 @@ NWMD-R010:
   description: new_model() supports `.path` containing a period
   tests:
   - BBR-NWMD-010
+NWMD-R011:
+    description: valid model extensions may be passed into new_model
```

Comment:
This is minor, but this reads to me like there is some extent to validate the extensions.  What about "new_model() ignores files extensions"?  (I suppose a similar comment can be made for the story description.)

Reply:
> hint that only .mod and .ctl would be accepted into new models.

I don't understand.  The current code just ignores all extensions.  What do we gain by not making this description match the behavior?